package-lock.json

```diff
@@ -2620,9 +2620,9 @@
       "license": "MIT"
     },
     "node_modules/@types/node": {
-      "version": "20.19.14",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-20.19.14.tgz",
-      "integrity": "sha512-gqiKWld3YIkmtrrg9zDvg9jfksZCcPywXVN7IauUGhilwGV/yOyeUsvpR796m/Jye0zUzMXPKe8Ct1B79A7N5Q==",
+      "version": "20.19.13",
```

Comment:
We probably don't want to include this change as part of this PR. This doesn't seem related to the Sidebar changes. Is there a reason we want or need to downgrade the node types package version?